doc/yay.8

```diff
@@ -118,8 +118,7 @@ Print current yay configuration\&.
 .PP
 \fB\-n \-\-numberupgrades\fR
 .RS 4
-Print number of packages that need to be updated\&. Note this does not perform
-a database refresh\&. Run \fByay \-Sy\fR Before this for an up to date result\&.
+Depreciated, use \fByay -Qu\fR and \fBwl -l\fR instead\%.
```

Comment:
sp: Deprecated

Reply:
Annnddddd error 500 when trying to make a PR:

![image](https://user-images.githubusercontent.com/16593899/42846872-162a988c-8a12-11e8-898b-a6a40ea2ff29.png)

Reply:
Ahhh rats! Just a few minutes too late.

Lovely error message that. :-o